src/electionguard/guardian.py

```diff
@@ -169,12 +176,25 @@ def __init__(
         self.id = id
         self.sequence_order = sequence_order
         self.set_ceremony_details(number_of_guardians, quorum)
-        self._backups_to_share = {}
-        self._guardian_election_public_keys = {}
-        self._guardian_election_partial_key_backups = {}
-        self._guardian_election_partial_key_verifications = {}
+        self._backups_to_share = {} if backups_to_share is None else backups_to_share
+        self._guardian_election_public_keys = (
+            {} if election_public_keys is None else election_public_keys
+        )
+        self._guardian_election_partial_key_backups = (
+            {} if partial_key_backups is None else partial_key_backups
+        )
+        self._guardian_election_partial_key_verifications = (
+            {}
+            if election_partial_key_verifications is None
+            else election_partial_key_verifications
+        )
 
-        self.generate_election_key_pair(nonce_seed if nonce_seed is not None else None)
+        if election_keys is None:
+            self.generate_election_key_pair(
+                nonce_seed if nonce_seed is not None else None
+            )
+        else:
+            self._election_keys = election_keys
```

Comment:
All of the code in this file addresses #355.  I was going to make that a separate PR, but it was substantially simpler to know it was working correctly by including it in this CLI command, hope that's ok.

Reply:
This seems like it should have been isolated in its own PR.

Reply:
Fair enough, will try to do better in the future